[2023-12-04T04:11:50.246+0000] {taskinstance.py:1159} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: catchup.Bash_2 scheduled__2023-12-03T00:00:00+00:00 [queued]>
[2023-12-04T04:11:50.258+0000] {taskinstance.py:1159} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: catchup.Bash_2 scheduled__2023-12-03T00:00:00+00:00 [queued]>
[2023-12-04T04:11:50.259+0000] {taskinstance.py:1361} INFO - Starting attempt 1 of 1
[2023-12-04T04:11:50.550+0000] {taskinstance.py:1382} INFO - Executing <Task(BashOperator): Bash_2> on 2023-12-03 00:00:00+00:00
[2023-12-04T04:11:50.668+0000] {standard_task_runner.py:57} INFO - Started process 120 to run task
[2023-12-04T04:11:50.676+0000] {standard_task_runner.py:84} INFO - Running: ['***', 'tasks', 'run', 'catchup', 'Bash_2', 'scheduled__2023-12-03T00:00:00+00:00', '--job-id', '175', '--raw', '--subdir', 'DAGS_FOLDER/catchup.py', '--cfg-path', '/tmp/tmp9rh1sszq']
[2023-12-04T04:11:50.721+0000] {standard_task_runner.py:85} INFO - Job 175: Subtask Bash_2
[2023-12-04T04:11:51.993+0000] {task_command.py:416} INFO - Running <TaskInstance: catchup.Bash_2 scheduled__2023-12-03T00:00:00+00:00 [running]> on host ec0adab034ec
[2023-12-04T04:11:53.195+0000] {taskinstance.py:1662} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='Dashrath' AIRFLOW_CTX_DAG_ID='catchup' AIRFLOW_CTX_TASK_ID='Bash_2' AIRFLOW_CTX_EXECUTION_DATE='2023-12-03T00:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2023-12-03T00:00:00+00:00'
[2023-12-04T04:11:53.202+0000] {subprocess.py:63} INFO - Tmp dir root location: /tmp
[2023-12-04T04:11:53.206+0000] {subprocess.py:75} INFO - Running command: ['/bin/bash', '-c', 'echo this is second task']
[2023-12-04T04:11:53.310+0000] {subprocess.py:86} INFO - Output:
[2023-12-04T04:11:53.332+0000] {subprocess.py:93} INFO - this is second task
[2023-12-04T04:11:53.333+0000] {subprocess.py:97} INFO - Command exited with return code 0
[2023-12-04T04:11:53.959+0000] {taskinstance.py:1400} INFO - Marking task as SUCCESS. dag_id=catchup, task_id=Bash_2, execution_date=20231203T000000, start_date=20231204T041150, end_date=20231204T041153
[2023-12-04T04:11:54.106+0000] {local_task_job_runner.py:228} INFO - Task exited with return code 0
[2023-12-04T04:11:54.614+0000] {taskinstance.py:2778} INFO - 0 downstream tasks scheduled from follow-on schedule check
